MAUI/DataGrid/selection.md

```diff
@@ -0,0 +1,547 @@
+
+{% tabs %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+public partial class MainPage : ContentPage
+{
+    public MainPage()
+    {
+        InitializeComponent();
+        ////Perform multiple selection using selected roes
+        var viewModel = new ViewModel();
+        dataGrid.SelectedRows.Add(viewModel.OrderInfoCollection[4]);
+        dataGrid.SelectedRows.Add(viewModel.OrderInfoCollection[6]);
+        dataGrid.SelectedRows.Add(viewModel.OrderInfoCollection[2]);
+        dataGrid.SelectedRows.Add(viewModel.OrderInfoCollection[11]);
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+![Multiple Selection in .NET MAUI DataGrid](Images/selection/maui-datagrid-multiple-selection.png)
+
+### Select all the rows
+You can select all the rows in the datagrid using [SfDataGrid.SelectAll()] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html#Syncfusion_Maui_DataGrid_SfDataGrid_SelectAll) method.
+{% tabs %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+public partial class MainPage : ContentPage
+{
+    public MainPage()
+    {
+        InitializeComponent();
+        dataGrid.SelectAll();
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+![All the rows selected in .NET MAUI DataGrid](Images/selection/maui-datagrid-selectall.png)
+
+## Keyboard behavior
+<table>
+<tr>
+<th>
+Key or KeyCombinations
+</th>
+<th>
+Description
+</th>
+</tr>
+<tr>
+<td>
+<kbd>DownArrow</kbd>
+</td>
+<td>
+Moves CurrentCell directly below the active current cell. If the CurrentCell is in last row, pressing <kbd>DownArrow</kbd> does nothing.
+</td>
+</tr>
+<tr>
+<td>
+<kbd>UpArrow</kbd>
+</td>
+<td>
+Moves the CurrentCell directly above the active current cell. If the CurrentCell is in first row, pressing <kbd>UpArrow</kbd> does nothing.
+</td>
+</tr>
+<tr>
+<td>
+<kbd>LeftArrow</kbd>
+</td>
+<td>
+Moves the current cell to previous to the active current cell. If the CurrentCell is in first cell, pressing <kbd>LeftArrow</kbd> does nothing. 
+</td>
+</tr>
+<tr>
+<td>
+<kbd>RightArrow</kbd>
+</td>
+<td>
+Moves the current cell to next to the active current cell. If the CurrentCell is in last cell, pressing <kbd>RightArrow</kbd> does nothing. 
+</td>
+</tr>
+<tr>
+<td>
+<kbd>Home</kbd> / <kbd> Ctrl</kbd> + <kbd>LeftArrow</kbd>
+</td>
+<td>
+Moves the current cell to the first cell of the current row.
+</td>
+</tr>
+<tr>
+<td>
+<kbd>End</kbd> / <kbd>Ctrl</kbd> + <kbd>RightArrow</kbd>
+</td>
+<td>
+Moves the current cell to the last cell of the current row.
+</td>
+</tr>
+<tr>
+<td>
+<kbd>PageDown</kbd>
+</td>
+<td>
+The SfDataGrid will be scrolled to next set of rows that are not displayed in view, including the row that are partially displayed and the current cell is set to last row.
+</td>
+</tr>
+<tr>
+<td>
+<kbd>PageUp</kbd>
+</td>
+<td>
+The SfDataGrid will be scrolled to previous set of rows that are not displayed in view, including the row that are partially displayed and the current cell is set to the first row.
```

Comment:
The SfDataGrid will be scrolled to the previous set of rows that are not displayed in the view, including the row that is partially displayed and the current cell is set to the first row.